apps/nextra/package.json

```diff
@@ -10,9 +10,7 @@
     "clean": "rimraf .next .turbo",
     "debug": "NODE_OPTIONS='--inspect' next dev",
     "dev": "next -p 3030",
-    "prebuild": "node nextra-remote-filepaths/fetch.js",
     "postbuild": "next-sitemap --config next-sitemap.config.mjs; node generate-language-sitemaps.cjs; node generate-sitemap-index.cjs",
-    "predev": "pnpm prebuild",
```

Comment:
I don't think these are supposed to be removed

Reply:
I chatted with andrew and he said is ok to remove @hariria

Reply:
ill take out and maybe move to separate pr

Reply:
Yeah its ok to remove